app/helpers/settings_schedule_helper.rb

```diff
@@ -0,0 +1,117 @@
+module SettingsScheduleHelper
+  private
+
+  def settings_schedule_summary(schedule, exp_table, object_class, object_name)
+    automation_request_check = schedule.sched_action[:method] == 'automation_request'
+    summary = [
+      settings_schedule_basic_info(schedule),
+      settings_schedule_filter_details(schedule, exp_table),
+    ]
+    if automation_request_check
+      summary.push(settings_schedule_object_details(schedule))
+      summary.push(settings_schedule_object_attribute(object_class, object_name))
+      summary.push(settings_schedule_attribute_value_pairs(schedule)) if schedule.filter[:ui][:ui_attrs].present?
+    end
+    safe_join(summary)
+  end
+
+  def settings_schedule_basic_info(schedule)
+    data = {
+      :title => _('Basic Information'),
+      :mode  => "settings_schedule_basic_info",
+    }
+
+    rows = [
+      row_data(_('Description'), schedule.description),
+      row_data(_('Active'), schedule.enabled),
+    ]
+
+    if schedule.sched_action[:method] && schedule.sched_action[:method] == "check_compliance"
+      rows.push(row_data(_('Action'), "#{ui_lookup(:model => schedule.resource_type)} #{_('Compliance Check')}"))
+    elsif schedule.sched_action[:method] && schedule.sched_action[:method] == "automation_request"
+      rows.push(row_data(_('Action'), _('Automate Task')))
+    else
+      rows.push(row_data(_('Action'), "#{ui_lookup(:model => schedule.resource_type)} #{_('Analysis')}"))
+    end
+
+    data[:rows] = rows
+    miq_structured_list(data)
+  end
+
+  def settings_schedule_filter_details(schedule, exp_table)
+    data = {
+      :title => _('Filter'),
+      :mode  => "settings_schedule_filter_details",
+    }
+
+    rows = []
+    unless schedule.sched_action[:method] == "automation_request"
+      if schedule.miq_search
+        search = schedule.miq_search
+        description = "Filter: #{search.description}"
+        filter_data = search.search_type == "user" ? "My #{description}" : "Global #{description}"
```

Comment:
Why are these two lines instead of one? Additionally these strings need to be externalized, something like: 

```
filter_data = search.search_type == "user" ? _("My Filter: ") + search.description : _("Global Filter: ") + search.description
```

Reply:
fixed